src/pages/Potato.tsx

```diff
@@ -0,0 +1,57 @@
+import axios, {AxiosError} from 'axios';
+import { useState } from 'react';
+
+type Food = {
+  fdcId: number;
+  description: string;
+  foodNutrients: Nutrient[];
+};
+
+type Nutrient = {
+  nutrientName: string;
+  value: number;
+  unitName: string;
+};
+
+const Potato = () => {
+  const [foods, setFoods] = useState<Food[]>([]);
+  
+  const fetchInfo = async () => {
+    try {
+      const fetchResult = await axios.get(`http://localhost:8080/api/usda/search`)
+      // fetchResult.data.data.foods is the array of foods
+      const foodList: Food[] = fetchResult.data.data.foods;
```

Comment:
What would happen if the fetchResult doesn’t return the expected structure. For example, if `fetchResult.data` is undefined or doesn't contain a `data.foods` array?

To keep it simple for now, how about we do something like:
```suggestion
      const foodList: Food[] = fetchResult?.data?.data?.foods ?? [];
```